src/test/csfle.rs

```diff
@@ -3436,6 +3436,69 @@ async fn bind(addr: &str) -> Result<TcpListener> {
     Ok(TcpListener::bind(addr.parse::<std::net::SocketAddr>()?).await?)
 }
 
+// Prose test 23. Range explicit encryption applies defaults
+#[tokio::test]
+async fn range_explicit_encryption_defaults() -> Result<()> {
+    let name = "range_explicit_encryption_defaults";
+    if !check_env(name, false) || !fle2v2_ok(name).await {
```

Comment:
```suggestion
    if !check_env(name, false) {
```

Suggest removing unneeded server checks. Test only creates local payloads.